jujugui/static/gui/src/test/test_env_api.js

```diff
@@ -420,15 +422,17 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       it('fires a login event on failed login', function() {
         let fired = false;
         let err;
-        env.on('login', evt => {
+        const listener = evt => {
           fired = true;
```

Comment:
This is a little scary as it's requiring an async event process to execute synchronously...these tests should all be upgraded to use `done()` but since this seams to work maybe we just do that as a future follow-up (with an issue created)